src/OpenApiRuntime/composer.json

```diff
@@ -21,13 +21,15 @@
         "php": ">=7.2",
         "jane-php/json-schema-runtime": "^5.0",
         "php-http/client-common": "^1.4 || ^2.0",
-        "php-http/httplug": "^1.0 || ^2.0",
-        "php-http/message-factory": "^1.0.2",
         "php-http/multipart-stream-builder": "^1.0",
         "symfony/options-resolver": "^3.1|^4.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^4.7|^5.5"
+        "phpunit/phpunit": "^4.7|^5.5",
+        "php-http/httplug": "^1.0 || ^2.0",
```

Comment:
As we will release this feature in a minor version (5.1), we can't remove `php-http/*` dependencies until next major